docassemble/mlhframework/data/questions/mlh_interview_framework.yml

```diff
@@ -831,3 +848,15 @@ subject: |
 content: |
   The lawyer's bar number is a five digit number after the letter P, usually listed 
   after the lawyer's name. For example: P12345.  
+---
+template: help_user_ask_role_template    # must edit/replace this if other roles apply
+subject: |
+  How do I know?
+content: |
+  You can find the parties by looking at the first page of an existing court document, like a complaint, petition, or answer. The top of the first page is the “caption” which has the basic case information. This area will list all of the parties in the case with their titles.
+
+  ${ image_plaintiff_or_defendant.show(width="100%") }
+
+  If you were the first person to file a form to start this case, you are the Plaintiff. If someone else started this case by filing in court, you
+  are the Defendant.
```

Comment:
@ekressmiller - was this new line intentional, or should "are the Defendant." continue on the prior line?

Reply:
unintentional, thanks!